src/osd/PeeringState.h

```diff
@@ -245,7 +245,91 @@ struct PeeringCtxWrapper {
   }
 };
 
-  /* Encapsulates PG recovery process */
+/**
+ * BackfillInterval
+ *
+ * Represents the objects in a range [begin, end)
+ *
+ * Possible states:
+ * 1) begin == end == hobject_t() indicates the the interval is unpopulated
+ * 2) Else, objects contains all objects in [begin, end)
+ */
+struct BackfillInterval {
```

Comment:
I'd like to stop adding things to PeeringState.h.  If possible, can we put this in another file?  Ideally with the backfill machinery?

Reply:
How about `peering_state_types.h` per analogy to `osd_types.h` and `bluestore_types.h`?

Reply:
Yeah, that would be fine.

Reply:
Moved to `recovery_types.h`.